### DISCUSSION TITLE
DNT v3.0 adds HTTP to MQTT as communication protocol option

### DISCUSSION THREAD

**Discussion by hdtodd** [merbanan/rtl_433#3043] [Show and tell]
DNT (Display Neighborhood Temperatures) v3.0 adds HTTP streaming as a communication protocol option to the previous MQTT option.  [http://github.com/hdtodd/DNT](http://github.com/hdtodd/DNT)

**Comment by gdt** [comment]
I know my views differ from the group centroid, but my take is that a python bridge script that reads syslog/json and does whatever some random data consumer wants is the right answer, and tht we should not try to support this in rtl-433 itself.   See examples/*relay*.py for inspiration; it shouldn't be hard.

**Comment by gdt** [comment]
I now see you are upstream.   You left out the link to the python libraries that talk to your server, in both regular and async flavors.
(Not really kidding; in the Home Assistant the world, the foo integration is almost always a thin HA wrapper around aiofoo.
